packager/common.gypi

```diff
@@ -122,6 +128,22 @@
           '-Werror',
         ],
       }],
+      ['static_link_binaries==1', {
+        'conditions': [
+          ['OS=="linux"', {
+            'defines': [
+              # Even when we are not using musl or uClibc, pretending to use
+              # uClibc on Linux is the only way to disable certain Chromium
+              # base features, such as hooking into malloc.  Hooking into
+              # malloc, in turn, fails when we are linking statically.
+              '__UCLIBC__',
```

Comment:
Is there side effects by defining `__UCLIBC__`, i.e. effects other than disabling malloc hooking?